### PR TITLE
[RFC][UN-8896] add tooltips on inputs, minor bug fix

### PR DESCRIPTION
**Pull request by telmaantunes**
https://uniplaces.atlassian.net/browse/UN-8896

to be merged after this https://github.com/uniplaces/ember-cli-uniq/pull/139

* textarea
screen shot 2017-04-27 at 15 35 20
screen shot 2017-04-27 at 15 35 25

* countries
screen shot 2017-04-27 at 15 24 18
screen shot 2017-04-27 at 15 24 24

* fixed a border radius bug here
screen shot 2017-04-27 at 15 40 12

### CODE REVIEW

**Comment by AndreJoaquim** [comment]
TOOLTIPS EVERYWHERE!

File: addon/styles/partials/_uni-mobile-number.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .uni-mobile-number {
   @include mobile-number();
+  display: inline-flex;
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-mobile-number.scss]:
```sass
@include inline-flex()
```

File: addon/styles/partials/_uni-textarea.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .uni-textarea {
   margin-bottom: 46px;
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-textarea.scss:2]: 👀

File: tests/dummy/app/templates/application.hbs [hunk 3]
```diff
@@ -474,9 +474,9 @@
     \{{uni-mobile-number}}
   </p>
 
-  {{uni-mobile-number preferredCountries=(array 'PT' 'DE' 'GB')}}
```
Comment (AndreJoaquim) [inline tests/dummy/app/templates/application.hbs]: Can you keep one without tooltip?

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

File: addon/styles/partials/_uni-textarea.scss (hunk 2 above)
Comment (telmaantunes) [reply addon/styles/partials/_uni-textarea.scss:2]: indeed 😿

**Review by AndreJoaquim — APPROVED** [review]

**Pull request merged**